client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -27,44 +27,44 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = () =>
 	const [ hasBackground, setHasBackground ] = useState( false );
 
 	return (
-		<Switch>
-			<Route exact path={ Step.IntentGathering }>
-				<div
-					className={ classNames( 'onboarding-block__acquire-intent', {
-						'has-background': hasBackground && siteVertical,
-					} ) }
-				>
-					{ ( hasBackground || siteVertical ) && (
-						<VerticalBackground id={ siteVertical?.id } onLoad={ () => setHasBackground( true ) } />
-					) }
-					<div className="onboarding-block__questions">
-						<h2 className="onboarding-block__questions-heading">
-							{ ! siteVertical &&
-								! siteTitle &&
-								NO__( "Let's set up your website – it takes only a moment." ) }
-						</h2>
-						<StepperWizard
-							stepComponents={ [ VerticalSelect, ( siteVertical || siteTitle ) && SiteTitle ] }
-						/>
-						{ siteVertical && (
-							<div className="onboarding-block__footer">
-								<Link
-									to={ Step.DesignSelection }
-									className="onboarding-block__question-skip"
-									isLink
-								>
-									{ /* @TODO: add transitions and correct action */ }
-									{ siteTitle ? NO__( 'Continue' ) : NO__( "Don't know yet" ) } →
-								</Link>
-							</div>
-						) }
+		<>
+			<VerticalBackground id={ siteVertical?.id } onLoad={ () => setHasBackground( true ) } />
```

Comment:
After-the-fact review, the `id` prop [is no longer used](https://github.com/Automattic/wp-calypso/pull/38577/files#diff-b2f89f38cdbc8bc40bc2f1ac96a58efbR24) by `VerticalBackground`, so we shouldn't pass it anymore. TypeScript wasn't complaining, since `id` is still listed in `VerticalBackground`'s props interface definition.

I'll take care of this.

Reply:
#38597